lib/Parse/ParseSIL.cpp

```diff
@@ -1516,8 +1516,8 @@ bool getApplySubstitutionsFromParsed(
   return false;
 }
 
-// FIXME: we work around canonicalization of PolymorphicFunctionType
-// by generating GenericSignature and transforming the input, output
+// We work around the canonicalization of PolymorphicFunctionType
+// by generating a GenericSignature and transforming the input, output
```

Comment:
@practicalswift could you add the FIXME back here (per @gottesmm's clarification), and then re-squash?